lib/SILOptimizer/Utils/SILIsolationInfo.cpp

```diff
@@ -1017,10 +1045,10 @@ void SILIsolationInfo::print(llvm::raw_ostream &os) const {
   }
 }
 
-bool SILIsolationInfo::hasSameIsolation(ActorIsolation actorIsolation) const {
+bool SILIsolationInfo::hasSameIsolation(ActorIsolation other) const {
   if (getKind() != Kind::Actor)
     return false;
-  return getActorIsolation() == actorIsolation;
+  return getActorIsolation() == other;
```

Comment:
I did this renaming since SILIsolationInfo has a field called actorIsolation. It can get confusing what one is referring to int he debugger.